doc/whats_new/v0.20.rst

```diff
@@ -108,6 +107,11 @@ Decomposition, manifold learning and clustering
 - Fixed a bug in :func:`datasets.fetch_kddcup99`, where data were not properly
   shuffled. :issue:`9731` by `Nicolas Goix`_.
 
+Model evaluation and meta-estimators
+
+- Fixed a bug in :func:`metrics.roc_auc_score`, where float calculations sometimes
```

Comment:
I think it can written a bit better, maybe something like:
```
Fixed bug due to floating point error in :func:`metrics.roc_auc_score` with non-integer sample weights. 
:issue:`9786` by :user:`Hanmin Qin <qinhanmin2014>`.
```